src/main/config/config.c

```diff
@@ -478,36 +478,6 @@ static void validateAndFixConfig(void)
     featureDisableImmediate(FEATURE_RSSI_ADC);
 #endif
 
-if (systemConfig()->configurationState == CONFIGURATION_STATE_UNCONFIGURED) {
-
-#ifdef USE_DASHBOARD
-    featureEnableImmediate(FEATURE_DASHBOARD);
-#endif
-#ifdef USE_LED_STRIP
-    featureEnableImmediate(FEATURE_LED_STRIP);
-#endif
-#ifdef USE_OSD
```

Comment:
For consideration:
```
#if defined(USE_OSD) && USE_OSD >= 2
```

It will allow config.h to `#define USE_OSD 2` and `make <target> OPTIONS='USE_OSD=2`

Reply:
Was thinking about missing default features from unified_targets and adding in config.h something like:
```
#define DEFAULT_FEATURES     (FEATURES_OSD)
```

Reply:
@haslinghuis : https://github.com/betaflight/betaflight/pull/13494/files#diff-384c3b23c60da197eb7ecccd9bbfd19d68d96245f45181ec4db541e9c1014b92R418-R432

All that is necessary is to define autoFeatures by target

Reply:
Updated and tested.